src/scss/_about.scss

```diff
@@ -2,16 +2,16 @@
   @include background-pattern;
   width: 100%;
   box-sizing: border-box;
-  margin: 0 0 10em; //bottom here margin to allow for the fixed image to show
+  overflow-x: hidden; //hide svg elements which extend to the right
```

Comment:
these can be condensed to `overflow-hidden` unless there is a reason to specify individually. If so please leave a comment explaining why.